### PR TITLE
Add icon for 365 DataScience

### LINKED ISSUE #10221: 365 DataScience
### Brand Name

365 DataScience

### Website

https://365datascience.com/

### Popularity Metric

https://www.similarweb.com/website/365datascience.com/#overview
global rank 50,000

### Official Resources for Icon and Color

image

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by BenHur28**
image

**Issue:** closes #10221

**Similarweb rank:** [~115k](https://www.similarweb.com/website/365datascience.com/#overview)

### Checklist

- ☑️ I updated the JSON data in `_data/simple-icons.json`
- ☑️ I optimized the icon with SVGO or SVGOMG
- ☑️ The SVG `viewbox` is `0 0 24 24`

### Description

Took color from main webpage https://365datascience.com/

### CODE REVIEW

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Thanks for taking this one on, @BenHur28 👍🏻

However, much as I would personally prefer to run with _just_ the icon, I think we are going to have to include the "365" as well - I can't find any examples of them using the icon on its own, outside of their favicon.

**Comment by BenHur28** [comment]
Hello @PeterShaggyNoble, 

I have added the "365" to the logo as you suggested. I also updated the initial preview image with a new preview image.

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Thanks, @BenHur28, but the new version is vertically stretched; it should be 24dp in _width_, with the height remaining relative to that, preserving the aspect ratio.

**Comment by BenHur28** [comment]
Changed the 365DataScience.svg to have 24px width while maintaining proper height aspect ratio. Updated preview image with new svg preview image.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -31,6 +31,11 @@
             "source": "https://1password.com/press",
             "guidelines": "https://1password.com/press"
         },
+        {
+            "title": "365 DataScience",
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:35]:
```suggestion
            "title": "365 Data Science",
```

File: icons/365datascience.svg [hunk 2]
```diff
@@ -0,0 +1 @@
....07 0 2.221.509 2.23 1.692a1.172 1.172 0 0 1-.786 1.087c.542.175.924.7.924 1.168 0 1.36-1.305 1.758-2.399 1.758S.008 15.704 0 14.442h1.078c0 .572.786.77 1.248.77.542 0 1.264-.222 1.264-.818 0-.333-.187-.746-1.208-.746zm5.008-1.43v.833c.251-.376.866-.587 1.312-.587 1.144 0 2.343.468 2.343 1.85 0 1.233-1.207 1.844-2.35 1.844s-2.368-.611-2.368-1.858v-2.075c0-1.24 1.168-1.82 2.335-1.82 1.233 0 2.327.477 2.327 1.645H9.043c0-.492-.628-.738-1.24-.738-.64-.008-1.289.334-1.289.937zm2.593 2.096c0-.612-.567-.937-1.28-.937-.715 0-1.323.318-1.314.937.01.619.608.908 1.305.908.673 0 1.289-.297 1.289-.908zm5.997-2.812h-3.063v.977h1.1c1.209 0 2.287.485 2.279 1.787 0 1.263-1.078 1.867-2.221 1.867-1.247 0-2.317-.437-2.326-1.74h1.064c0 .613.713.77 1.248.77s1.174-.238 1.174-.897c.009-.628-.628-.81-1.216-.81H10.99v-2.94H15.1zm7.917-3.69a.95.95 0 0 0-.456 1.78l-1.925 4.72a.944.944 0 0 0-.8.082l-2.046-2.342a.943.943 0 0 0-.09-1.27.943.943 0 0 0-1.41 1.24.943.943 0 0 0 1.25.25l2.045 2.343a.952.952 0 0 0-.208.593.95.95 0 0 0 .266.693.95.95 0 0 0 1.57-.32.95.95 0 0 0 .062-.373.943.943 0 0 0-.343-.73l1.95-4.772a.826.826 0 0 0 .136.01.95.95 0 0 0 .908-.578.95.95 0 0 0-.214-1.054.95.95 0 0 0-.694-.266z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/365datascience.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>365 Data Science</title><path d="M1.5 13.647v-.858h.818c.575 0 1.183-.152 1.19-.668 0-.318-.258-.738-1.166-.738-.502 0-1.183.174-1.183.683H.09c0-1.2 1.216-1.633 2.26-1.633 1.07 0 2.221.509 2.23 1.692a1.172 1.172 0 0 1-.786 1.087c.542.175.924.7.924 1.168 0 1.36-1.305 1.758-2.399 1.758S.008 15.704 0 14.442h1.078c0 .572.786.77 1.248.77.542 0 1.264-.222 1.264-.818 0-.333-.187-.746-1.208-.746zm5.008-1.43v.833c.251-.376.866-.587 1.312-.587 1.144 0 2.343.468 2.343 1.85 0 1.233-1.207 1.844-2.35 1.844s-2.368-.611-2.368-1.858v-2.075c0-1.24 1.168-1.82 2.335-1.82 1.233 0 2.327.477 2.327 1.645H9.043c0-.492-.628-.738-1.24-.738-.64-.008-1.289.334-1.289.937zm2.593 2.096c0-.612-.567-.937-1.28-.937-.715 0-1.323.318-1.314.937.01.619.608.908 1.305.908.673 0 1.289-.297 1.289-.908zm5.997-2.812h-3.063v.977h1.1c1.209 0 2.287.485 2.279 1.787 0 1.263-1.078 1.867-2.221 1.867-1.247 0-2.317-.437-2.326-1.74h1.064c0 .613.713.77 1.248.77s1.174-.238 1.174-.897c.009-.628-.628-.81-1.216-.81H10.99v-2.94H15.1zm7.917-3.69a.95.95 0 0 0-.456 1.78l-1.925 4.72a.944.944 0 0 0-.8.082l-2.046-2.342a.943.943 0 0 0-.09-1.27.943.943 0 0 0-1.41 1.24.943.943 0 0 0 1.25.25l2.045 2.343a.952.952 0 0 0-.208.593.95.95 0 0 0 .266.693.95.95 0 0 0 1.57-.32.95.95 0 0 0 .062-.373.943.943 0 0 0-.343-.73l1.95-4.772a.826.826 0 0 0 .136.01.95.95 0 0 0 .908-.578.95.95 0 0 0-.214-1.054.95.95 0 0 0-.694-.266z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Thanks, @BenHur28. But I'm seeing some hairline differences when comparing against the logo in the website's header, most notably in 
[the top of the left circle](https://github.com/simple-icons/simple-icons/assets/15157491/3ec4d130-4694-4a29-8d15-a27a2c37ebf0) (with yours in red). Most likely, that can be solved by increasing the precision by 1 when optimising. Would you mind having another look at it, please?

Also, you're missing a space in the name - see below.

**Comment by BenHur28** [comment]
Increased the optimization precision by 1 and added space in between data and science as suggested.

**Comment by PeterShaggyNoble** [comment]
@BenHur28, it looks you may have forgotten to push the SVG with the updated, higher precision path here.

**Comment by BenHur28** [comment]
Hello @PeterShaggyNoble, I've checked the 365datascience.svg files and it does appear to be the version of the file where I had increased the precision by 1. Please check again and worse case I can recreate the svg file once more if needed.

**Comment by PeterShaggyNoble** [comment]
🤔 Dunno what's happening here, @BenHur28, but it looks like something somewhere along the way is reducing the precision to 3 on you.

I gave it a try myself and came up with the following path, if you'd like to run with it:
```
M1.4995 13.6448v-.8577h.8188c.5752 0 1.183-.152 1.1905-.6681 0-.3178-.2587-.7385-1.1666-.7385-.5024 0-1.183.1746-1.183.6832h-1.07c0-1.1993 1.2156-1.6326 2.2605-1.6326 1.07 0 2.2204.5086 2.2291 1.6916a1.1716 1.1716 0 0 1-.7861 1.0876c.5425.1746.9243.6995.9243 1.168 0 1.3588-1.3049 1.7581-2.3987 1.7581-1.0938 0-2.3108-.4345-2.3183-1.6979h1.0775c0 .5727.7862.7711 1.2483.7711.5426 0 1.2647-.2223 1.2647-.8188 0-.3328-.1871-.746-1.2081-.746zm5.0083-1.4304v.8339c.2512-.3768.8666-.5878 1.3124-.5878 1.1428 0 2.3422.4685 2.3422 1.8512 0 1.232-1.207 1.8436-2.3497 1.8436-1.1429 0-2.3673-.6116-2.3673-1.8587V12.222c0-1.2395 1.168-1.8184 2.3346-1.8184 1.232 0 2.3259.476 2.3259 1.6439H9.0422c0-.4923-.628-.7385-1.2396-.7385-.6405-.0075-1.2885.3341-1.2885.9369zm2.5934 2.0973c0-.6116-.5677-.9369-1.281-.9369-.7133 0-1.3212.3177-1.3124.9369.0088.6191.6079.908 1.3049.908.6718 0 1.2885-.2964 1.2885-.908zm5.9967-2.8119h-3.063v.977h1.1013c1.2082 0 2.2857.4848 2.2781 1.7872 0 1.2633-1.0787 1.8674-2.2216 1.8674-1.247 0-2.317-.437-2.3258-1.7406h1.0637c0 .6129.7133.771 1.2483.771.535 0 1.1742-.2385 1.1742-.8978.0088-.628-.6279-.81-1.2156-.81h-2.1488v-2.94h4.1092zm7.9169-3.6897a.9494.9494 0 0 0-.4559 1.7808l-1.9252 4.7183a.9444.9444 0 0 0-.7988.0829l-2.047-2.3422a.9432.9432 0 0 0-.089-1.27.9431.9431 0 0 0-1.4105 1.2412.943.943 0 0 0 1.2483.2498l2.0458 2.3422a.9518.9518 0 0 0-.2072.5927.9491.9491 0 0 0 .2654.6936.9493.9493 0 0 0 1.5701-.3198.9491.9491 0 0 0 .062-.3738.9432.9432 0 0 0-.3427-.7296l1.949-4.7723a.8264.8264 0 0 0 .137.0101.9493.9493 0 0 0 .9083-.5775.9493.9493 0 0 0-.2148-1.0547.9495.9495 0 0 0-.6936-.2654Z
```

**Review by service-paradis — APPROVED** [review]
Thanks @BenHur28 and @PeterShaggyNoble 🎉

**Pull request merged**